### DISCUSSION TITLE
Verified Commits for Github Codespaces

### DISCUSSION THREAD

**Discussion by Chaostheorie** [community/community#4051] [Codespaces] [answered]
Thank you for your work on codespaces. It's a neat integration and very useful for quickly testing a change without having to clone the full repository and set up a dev environment.

Though after using it for a recent change on a PR I noticed the commits coming from a codespace are not 'verified commits' (see the latest commit [here](https://github.com/python-gino/gino/pull/783)). The other commits were made with the web interface directly, using the 'edit file' function, and are as such marked as verified.

My question is then, out of curiosity, why are GitHub Codespace commits not marked as verified? Is there any concrete reason to not set the codespace up with this configuration or have I forgotten to configure something for my account?

Thanks in advance for any feedback on this question.

**Comment by 2percentsilk** [comment, marked as answer]
Hi there!

To set up verified commits, check out our GPG documentation at https://docs.github.com/en/codespaces/managing-your-codespaces/managing-gpg-verification-for-codespaces

**Reply by Chaostheorie** [reply]
Thank you for the link. That solves the problem
